tests/check_source_tests.py

```diff
@@ -69,7 +69,59 @@ def test_devel_project(self):
         """Accepts a request coming from a devel project"""
         self._setup_devel_project()
 
-        req_id = self.wf.create_submit_request(SRC_PROJECT, 'blowfish').reqid
+        req_id = self.wf.create_submit_request(SRC_PROJECT, 'blowfish', add_commit = False).reqid
+
+        self.assertReview(req_id, by_user=(self.bot_user, 'new'))
+
+        self.review_bot.set_request_ids([req_id])
+        self.review_bot.check_requests()
+
+        self.assertReview(req_id, by_user=(self.bot_user, 'accepted'))
```

Comment:
JFYI: you could save some lines by using `assertReviewScript` (you first need to assign `self.script = 'check_source.py`) or `assertReviewBot` (you need to setup the bot using `setup_review_bot` instead of manually... which would also save you some lines in the tear up).